src/main/java/uk/gov/hmcts/reform/juddata/scheduler/JrdUserProfileDataLoadStarter.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.juddata.scheduler;
+
+import static uk.gov.hmcts.reform.juddata.camel.util.MappingConstants.JUDICIAL_USER_PROFILE_ORCHESTRATION;
+import static uk.gov.hmcts.reform.juddata.camel.util.MappingConstants.ORCHESTRATED_ROUTE;
+
+import javax.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.CamelContext;
+import org.apache.camel.ProducerTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.juddata.camel.route.ParentOrchestrationRoute;
+import uk.gov.hmcts.reform.juddata.camel.service.AuditProcessingService;
+import uk.gov.hmcts.reform.juddata.camel.util.DataLoadUtil;
+
+@Component
+@Slf4j
+public class JrdUserProfileDataLoadStarter {
+
+    @Autowired
+    CamelContext camelContext;
+
+    @Autowired
+    ParentOrchestrationRoute parentOrchestrationRoute;
+
+    @Autowired
+    ProducerTemplate producerTemplate;
+
+    @Value("${start-route}")
+    private String startRoute;
+
+    @Autowired
+    DataLoadUtil dataLoadUtil;
+
+    @Autowired
+    AuditProcessingService schedulerAuditProcessingService;
+
+    @PostConstruct
+    public void postConstruct() throws Exception {
+        camelContext.start();
+        camelContext.getGlobalOptions().put(ORCHESTRATED_ROUTE, JUDICIAL_USER_PROFILE_ORCHESTRATION);
+        parentOrchestrationRoute.startRoute();
```

Comment:
since u have added it as a step in the job config.. do yu need to set the route name here?